hack/dockerfile/install/dockercli.installer

```diff
@@ -8,7 +8,7 @@ install_dockercli() {
 
 	arch=$(uname -m)
 	# No official release of these platforms
-	if [ "$arch" != "x86_64" ] && [ "$arch" != "s390x" ] && [ "$arch" != "armhf" ]; then
+	if [ "$arch" != "x86_64" ] && [ "$arch" != "s390x" ] && [ "$arch" != "aarch64" ] && [ "$arch" != "armhf" ]; then
```

Comment:
I think this was there because Docker 17.06 did not have pre-built binaries for aarch64;

<img width="910" alt="Screenshot 2021-02-17 at 16 47 55" src="https://user-images.githubusercontent.com/1804568/108229306-f9a2b380-713f-11eb-9275-38afcd6bf8f9.png">